python_on_whales/docker_client.py

```diff
@@ -102,6 +109,7 @@ def __init__(
         compose_compatibility: Optional[bool] = None,
         client_binary: str = "docker",
         client_call: List[str] = ["docker"],
+        client_type: Literal["docker", "podman", "nerdctl", "unknown"] = "unknown",
```

Comment:
Is this intended as a public attribute of the class? (it would be helpful if it was). If so does it need adding to docs somewhere?

Reply:
It could be in the future. For the moment, it's not necessary for the goal of this PR.